internal/engine/compiler/compiler_post1_0_test.go

```diff
@@ -238,6 +245,65 @@ func TestCompiler_compileMemoryCopy(t *testing.T) {
 	}
 }
 
+func requireBenchmarkNoError(b *testing.B, err error) {
```

Comment:
renaming will be a little less distracting as it looks more like the assertion and still won't name conflict. please move this to the bottom of the file.

```suggestion
func requireNoError(b *testing.B, err error) {
```